notebooks/All About REM Statistics.ipynb

```diff
@@ -17,19 +17,20 @@
     "We do this because we generally don't know everything about each\n",
     "home. Rewiring America has a large database, based on public records,\n",
     "that tells us things like when a home what built and how many square feet\n",
-    "it is. But it does not always give us all the details that would help\n",
-    "us estimate savings. So we look at other similar homes nearby that we know\n",
-    "more about, and use statistics across all of them to decide what is likely\n",
+    "it is. But it does not give us all the details that we need to accurately\n",
+    "predict savings. So we instead use a set of simulated homes that we know\n",
+    "more about, predict the energy consumption for each of them, \n",
+    "and compute statistics across the set of homes to decide what is likely\n",
     "to happen in the given home.\n",
```

Comment:
Sure. If you think there is specific language I missed then by all means bring it in.

Reply:
I just added a more detail on how we get this set of samples from a conditional probability distribution because I think that is needed to really understand what is happening, particularly given the statistics I added about 89/200 samples having fuel oil water heating.